llvm-runtimes/compiler-rt/compiler-rt-20.0.0_pre20241215.ebuild

```diff
@@ -110,6 +110,7 @@ src_configure() {
 		-DCOMPILER_RT_BUILD_PROFILE=OFF
 		-DCOMPILER_RT_BUILD_SANITIZERS=OFF
 		-DCOMPILER_RT_BUILD_XRAY=OFF
+		-DCOMPILER_RT_BUILTINS_HIDE_SYMBOLS=OFF
```

Comment:
Err, when you were talking to me of "GNU extensions", I didn't think you were talking of visibility changes. I need to think about it more, since I think it will affect programs compiled with compiler-rt.

Reply:
If we want to keep hiding symbols for the main compiler-rt build, an another option would be building a separate copy of compiler-rt with `-DCOMPILER_RT_BUILTINGS_HIDE_SYMBOLS=OFF` in the libgcc ebuild.

Reply:
Hmm... actually, I've just tried using `llvm-runtimes/compiler-rt-19.1.4::gentoo` (which hides symbols) and installing only `llvm-runtimes/libgcc-19.1.4` from this PR / my overlay and... it works. :exploding_head: As in - the package builds and my local rustup toolchains work.

I think [the linker script](https://github.com/llvm/llvm-project/blob/main/llvm-libgcc/gcc_s.ver.in) would complain if there were symbols missing in compiler-rt (especially since I'm using LLD). I also checked if all the symbols referenced by it are present in the builtins ([here is the output of nm](https://gist.github.com/vadorovsky/a0971031c830072049c659182b87c123)) and looks like the answer is **yes** - even `T __gcc_personality_v0` is there.

Let me double check on more machines and on a clean container. I will drop the compiler-rt change and push once I feel confident.

Reply:
Yeah, it works well, pushed. Sorry for introducing that change in the first place - I took this part of CMake config for granted:

https://github.com/llvm/llvm-project/blob/llvmorg-19.1.6/llvm-libgcc/CMakeLists.txt#L92

But it really looks like it's unnecessary

Reply:
Thanks. Still, I'm going to check symbol visibility later (i.e. compare it with libgcc).